airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/pipeline.py

```diff
@@ -31,29 +34,73 @@ def get_bumped_version(version: str, bump_type: str) -> str:
     return str(new_version)
 
 
+class RestoreVersionState(Step):
+    context: ConnectorContext
+
+    title = "Restore original version state"  # type: ignore
+
+    def __init__(self, context: ConnectorContext) -> None:
+        super().__init__(context)
+        connector = context.connector
+        if connector.metadata_file_path.is_file():
+            self.metadata_content = connector.metadata_file_path.read_text()
+        if connector.dockerfile_file_path.is_file():
```

Comment:
I wonder if when we merge all these upgrade PRs we would be happy to remove airbyte-ci code paths that work with Dockerfiles. I'm not positive we can — some custom connectors or java aconnectors would perhaps still want them.

But at least, we would be able to delete codepaths that work with `setup.py` and that would be so cool!

Reply:
Yep, to give an answer to @natikgadzhi : 
This is something we can eventually do once this `up_to_date` command is productionized and automated